db/db_iter.cc

```diff
@@ -82,6 +82,7 @@ class DBIter: public Iterator {
     if (kForward==direction_)
     {
       ParsedInternalKey parsed;
+      parsed.type=kTypeValue; parsed.sequence=0; parsed.expiry=0; // clear warnings
```

Comment:
The "clear warnings" comment doesn't tell me enough. Is this to avoid compiler warnings about using uninitialized data members? If so, it would be cleaner to initialize the value-type data members in the ParsedInternalKey constructor, which is already defined in dbformat.h.